### PR TITLE
Use prefix for Token storage

### PR DESCRIPTION
**Pull request by guperrot**
Please have a look at our [guidelines for contributions](https://github.com/Microsoft/AppCenter-SDK-Android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Use prefix for Token storage preferences like in every module. To avoid collisions.

## Related PRs or issues

[AB#60030](https://msmobilecenter.visualstudio.com/web/wi.aspx?pcguid=27bf89de-9697-418a-accf-1aa125b22914&id=60030)

### CODE REVIEW

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java [hunk 1]
```diff
@@ -185,7 +187,7 @@ public void canReadTokenFromCacheWhenTokenValid() {
                 .withDbAccount("dbAccount")
                 .withDbCollectionName("collection")
                 .withToken(FAKE_TOKEN));
-        when(SharedPreferencesManager.getString(READONLY_PARTITION_NAME)).thenReturn(tokenResult);
+        when(SharedPreferencesManager.getString(PREFERENCE_PARTITION_PREFIX + READONLY_PARTITION_NAME)).thenReturn(tokenResult);
```
Comment (YiMysty) [inline sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: what is `PREFERENCE_PARTITION_NAMES`?

Comment (YiMysty) [reply sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: READONDLY partition is readonly partition

File: sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Constants.java [hunk 2]
```diff
@@ -26,37 +26,47 @@
...
      */
-    static final String PARTITION_NAMES = "partitions";
+    static final String PREFERENCE_PARTITION_NAMES = PREFERENCE_PREFIX + "partitions";
 
     /**
-     * Name of the service.
+     * Cached partition names list file name.
      */
-    static final String SERVICE_NAME = "Storage";
+    static final String PREFERENCE_PARTITION_PREFIX = PREFERENCE_PREFIX + "partition.";
```
Comment (YiMysty) [inline sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Constants.java]: extra .

File: sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java (hunk 1 above)
Comment (guperrot) [reply sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java]: Please read description of the issue.

**Pull request merged**